view/templates/admin/blocklist/server.tpl

```diff
@@ -4,20 +4,23 @@
 	}
 </script>
 <div id="adminpage">
-	<h1>{{$title}} - {{$page}}</h1>
+	<h2>{{$title}} - {{$page}}</h2>
 	<p>{{$intro}}</p>
-	<p>{{$public}}</p>
+	<p>{{$public nofilter}}</p>
+	{{$syntax nofilter}}
 
-	<h2>{{$addtitle}}</h2>
+	<h3>{{$addtitle}}</h3>
```

Comment:
Same as what @tobiasd had said above. We have to start with ```<h1>``` at all the time and go down without a missing level. This is one of the rules for better accessibility.

Reply:
See https://github.com/friendica/friendica/pull/7431#discussion_r307771143